bees/mixcloudbee/mixcloudbee.go

```diff
@@ -0,0 +1,133 @@
+/*
+ *    Copyright (C) 2018 Stefan Derkits
+ *                  2018 Christian Muehlhaeuser
+ *
+ *    This program is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU Affero General Public License as published
+ *    by the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *    Authors:
+ *      Stefan Derkits <stefan@derkits.at>
+ *      Christian Muehlhaeuser <muesli@gmail.com>
+ */
+
+package mixcloudbee
+
+import (
+	"time"
+
+	"github.com/horrendus/go-mixcloud"
+
+	"github.com/muesli/beehive/bees"
+)
+
+// MixcloudBee is a Bee that can interface with Mixcloud.
+type MixcloudBee struct {
+	bees.Bee
+
+	baseUrl string
+	feed    string
+
+	client     *mixcloud.Client
+	lastUpdate time.Time
+
+	eventChan chan bees.Event
+}
+
+// Poll a Mixcloud Cloudcasts feed
+func (mod *MixcloudBee) pollFeed(feed string) {
+	mod.Logln("Parsing feed")
```

Comment:
A _bit_ nitpicky, but this logging statement could probably be better served with a mod.LogDebugf or similar, as to not clutter the main output. @muesli can confirm whether or not I'm right in making this assessment 😄